modules/storage-ephemeral-storage-diagnosing.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * storage/application-ephemeral-storage.adoc
+
+[id='diagnosing-evicted-pod-{context}']
+= Diagnosing an evicted pod
+
+{product-title} might evict a pod from its node when the node’s ephemeral 
+storage is exhausted. Depending on the extent of ephemeral storage 
+exhaustion, the eviction might or might not be graceful. In graceful 
+eviction, the main process, PID 1, of each container receives a 
+SIGTERM signal, then some time later, a SIGKILL signal if the process 
+is still running. In non-graceful eviction, the main process of each 
+container immediately receives a SIGKILL signal.
+
+To get a list of all pods so you can review their statuses:
+
+----
+$ oc get pod test
+NAME      READY     STATUS    RESTARTS   AGE
+test      0/1       Evicted   0          1m
+
+$ oc get pod test -o yaml
+...
+status:
+  message: 'Pod The node was low on resource: [DiskPressure].'
+  phase: Failed
+  reason: Evicted
+----
+
+An evicted pod has phase *Failed* and reason *Evicted*. The evicted pod is 
+not restarted, regardless of the value of *restartPolicy*. However, 
```

Comment:
If the replication controller's going to make a new one, why do I need to check for evicted ones? What do I need to do to fix the evicted pods?